### DISCUSSION TITLE
Disabling the "Just Chat" function on Lobe Chat.

### DISCUSSION THREAD

**Discussion by baptistechretien** [lobehub/lobehub#4978] [General Question]
Hello,

I would like to know how to disable the "Just Chat" feature on Lobe Chat. Could you please guide me through the procedure?

Thank you in advance for your help.

Best regards,

**Comment by arvinxx** [comment]
not allow

**Reply by baptistechretien** [reply]
Hello,

Thank you for your response. Could you please let me know if it would be possible to add a feature to disable "Just Chat" on Lobe Chat via the environment variable Feature Flags (FEATURE_FLAGS)?

Best regards,

**Reply by arvinxx** [reply]
No. It's not possible to do it.

**Comment by m0rtyn** [comment]
> I would like to know how to disable the "Just Chat" feature on Lobe Chat. Could you please guide me through the procedure?

I believe this request might be more about hiding "Just Chat" from the initial view when the application loads, rather than completely disabling its functionality. 

Right now, as I know, there's no direct way to hide it on startup, but it is possible with User JS/CSS (as a workaround).

Beyond that, I think it'd be awesome if we had a proper setting to let users reorder "Just Chat" – move it off the top, for instance. That would be a great customization feature!
